### DISCUSSION TITLE
Web-fancyapps | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#343] [Announcements]
# Web-fancyapps | Zi-Zi's Journey

Robust JavaScript UI Component Library.

https://gz-metal-cell.github.io/2024/07/22/Web-fancyapps/

**Comment by promefire** [comment]
有点帅哦

**Reply by GZ-Metal-Cell** [reply]
太麻烦了😭开发成本太高了😭不好用😭
